posts/2021-10-29-kube-secrets-210011.adoc

```diff
@@ -114,7 +114,7 @@ Create `features-bom` (bill of Materials) file for the user feature. The `featur
 </project>
 ----
 
-Generate the `features.json` file with Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information found within a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any liberty features(s) from a maven repository. Provide the maven coordinate of the custom made `features-bom` file.
+Generate the `features.json` file with Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information found within a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any Liberty features(s) from a Maven repository. Provide the Maven coordinate of the custom made `features-bom` file.
```

Comment:
suggested edit:
Generate the `features.json` file with the Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information from a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any Liberty feature from a Maven repository. Provide the Maven coordinate of the custom made `features-bom` file.